.github/workflows/test-expand.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
https://www.shellcheck.net

I think we should at least double-quote variables.

[SC2086](https://github.com/koalaman/shellcheck/wiki/SC2086)